src/vsg/utils/GraphicsPipelineConfigurator.cpp

```diff
@@ -471,8 +471,8 @@ bool GraphicsPipelineConfigurator::assignBuffer(const std::string& name, ref_ptr
 /// deprecated. use GraphicsPipelineConfigurator::assignBuffer() instead
 bool GraphicsPipelineConfigurator::assignUniform(const std::string& name, ref_ptr<Data> data)
 {
-    warn("GraphicsPipelineConfigurator::enableUniform() has been deprecated."
-         " use GraphicsPipelineConfigurator::enableBuffer() instead.");
+    warn("GraphicsPipelineConfigurator::assignUniform() has been deprecated."
```

Comment:
Is the warning redundant if  we are using the [[deprecated]] compiler hint?

Reply:
Possibly. I don't know how this attribute and how VSG's logging plays with release and debug. I figured you might have some opinion on which is best. I can take this out if you think it best.

Reply:
Once this PR is settled a bit further I'll merge this PR as a branch and then do testing with it.

Reply:
for deprecations, I believe it best to be obnoxious about it. Otherwise it's easier to ignore.